src/cpp/r/R/Api.R

```diff
@@ -3,6 +3,85 @@
    invisible(.rs.restartR(command))
 })
 
+.rs.addApiFunction("initializeProject", function(path = getwd())
+{
+   path <- .rs.ensureScalarCharacter(path)
+   
+   # if this is an existing file ...
+   if (file.exists(path))
+   {
+      # ... and it's an .Rproj file, then just return that (don't 
+      # re-initialize the project)
+      if (grepl("[.]Rproj$", path))
+         return(.rs.normalizePath(path, winslash = "/"))
+      
+      # ... and it's not a directory, bail
+      if (!utils::file_test("-d", path))
+      {
+         fmt <- "can't create project at path '%s'"
```

Comment:
Nit: Could we be more explicit about why we can't create the project here? (i.e. the user passed a file but we expected a directory)